doc-content/enterprise-only/installation/clustering-kie-server-failover-proc.adoc

```diff
@@ -0,0 +1,106 @@
+[id='clustering-kie-server-failover-proc_{context}']
+= Failover for jobs in the RUNNING state on failed {KIE_SERVER} nodes
+
+By default, when multiple {KIE_SERVER} nodes are running in cluster, if one {KIE_SERVER} node fails, other available {KIE_SERVER} nodes pick up any jobs in the `QUEUED` or `RETRYING` state from the failed {KIE_SERVER} node. However, these nodes do not pick up jobs that are in the `RUNNING` state.
+
+You can configure your cluster to failover all jobs when a {KIE_SERVER} node fails, including jobs in the `RUNNING` state, to be requeued to another {KIE_SERVER} node. If there is timer start node in a process on the failed {KIE_SERVER} node, the process is started on only one of the nodes.
```

Comment:
"You can configure your cluster to failover all jobs when a {KIE_SERVER} node fails, including jobs in the `RUNNING` state, to be requeued to another {KIE_SERVER} node. If there is timer start node in a process on the failed {KIE_SERVER} node, the process is started on only one of the nodes."

Possible rewrite:
You can configure your cluster to include jobs in the `RUNNING` state, to be requeued to another {KIE_SERVER} node. If there is timer start node in a process on the failed {KIE_SERVER} node, the process is started on only one of the nodes.